web-local/src/lib/components/metrics-definition/MetricsDefinitionExploreMetricsButton.svelte

```diff
@@ -38,7 +38,12 @@
   ) {
     buttonDisabled = true;
     buttonStatus = "MISSING_MODEL_OR_TIMESTAMP";
-  } else if (measures?.length === 0 || dimensions?.length === 0) {
+  } else if (
+    // check if all the measures have a valid expression
+    measures?.filter((measure) => measure?.expression?.length)?.length === 0 ||
+    // and if the dimensions all have a valid property
+    dimensions?.filter((dimension) => dimension?.property?.length)?.length === 0
```

Comment:
Likewise, this doesn't close the parentheses

Reply:
also what the linter does

Reply:
Ah nevermind, I just misread it and didn't see one of the parentheses